src/Numerics/ODESolvers/SplitExplicitMethod.jl

```diff
@@ -0,0 +1,192 @@
+export SplitExplicitSolver
+
+using ClimateMachine.DGMethods:
+    initialize_states!,
+    tendency_from_slow_to_fast!,
+    cummulate_fast_solution!,
+    reconcile_from_fast_to_slow!
+
+LSRK2N = LowStorageRungeKutta2N
+
+"""
+    SplitExplicitMethod(slow_solver, fast_solver; dt, t0 = 0)
+
+This is a time stepping object for explicitly time stepping the differential
+equation given by the right-hand-side function `f` with the state `Q`, i.e.,
+
+```math
+  \\dot{Q_fast} = f_fast(Q_fast, Q_slow, t)
+  \\dot{Q_slow} = f_slow(Q_slow, Q_fast, t)
+```
+
+with the required time step size `dt` and optional initial time `t0`.  This
+time stepping object is intended to be passed to the `solve!` command.
+
+This method performs an operator splitting to timestep the vertical average of the model at a faster rate than the full model. This results in a first-order time stepper.
+
+### References
+"""
+mutable struct SplitExplicitSolver{SS, FS, RT} <: AbstractODESolver
+    "slow solver"
+    slow_solver::SS
+    "fast solver"
+    fast_solver::FS
+    "time step"
+    dt::RT
+    "time"
+    t::RT
+    "param to couple the models"
+    coupled::Bool
```

Comment:
I don't understand the `coupled` parameter.

Reply:
its to turn on and off the custom functions we're adding to pass back and forth info. currently those functions are nothing, but after I add them in the next PR I want to make sure that the timestepper still properly steps two uncoupled models